src/components/Semester/OrderDropdown.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div class="dropdown">
+    <button class="dropdown-button" data-cyId="dropdown-button">
+      <img class="dropdown-button--image" />
+      View
+    </button>
+    <div class="dropdown-content">
+      <order-dropdown-option
+        :selected="!compact"
+        image="/src/assets/images/schedule-view/view-settings/default-sem.svg"
+        label="Default"
+        @click.stop="$emit('click-compact', false)"
+      />
+      <order-dropdown-option
+        :selected="compact"
+        image="/src/assets/images/schedule-view/view-settings/four-column.svg"
+        label="Compact"
+        @click.stop="$emit('click-compact', true)"
+      />
+      <div class="dropdown-content--hline" />
+      <order-dropdown-option
+        :selected="orderByNewest"
+        image="/src/assets/images/schedule-view/view-settings/newest-arrow.svg"
+        label="Newest"
+        @click.stop="onOrderClick(true)"
```

Comment:
Is there a reason you used `@click.stop` instead of just `@click`? Not sure we ever do this elsewhere.